### PR TITLE
[Concept Entry] Sklearn: Linear Discriminant Analysis

### LINKED ISSUE #5306: [Concept Entry] Sklearn Linear Discriminant Analysis Linear Discriminant Analysis
### Reason/inspiration (optional)

We would like a new entry on the `Linear Discriminant Analysis` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/linear-discriminant-analysis/linear-discriminant-analysis.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Sriparno08**
### Description

Added a new term entry on `Linear Discriminant Analysis` in Sklearn.

### Issue Solved

Closes #5306

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Sriparno08, thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

File: content/sklearn/concepts/linear-discriminant-analysis/linear-discriminant-analysis.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+In Sklearn, **Linear Discriminant Analysis (LDA)** is a supervised algorithm that aims to project data onto a lower-dimensional space while preserving the information that discriminates between different classes. LDA finds a set of directions in the original feature space that maximize the separation between the classes. These directions are called discriminant directions. By projecting the data onto these directions, LDA reduces the dimensionality of the data while retaining the information that is most relevant for classification.
+
+## Syntax
+
+```pseudo
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+
+# Create an LDA model
+model = LinearDiscriminantAnalysis()
```
[the remark this answers is not in the document]
Comment (Sriparno08) [reply content/sklearn/concepts/linear-discriminant-analysis/linear-discriminant-analysis.md]: Yup, we can certainly do that.

**Comment by Sriparno08** [comment]
Made the changes, @mamtawardhani.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @Sriparno08  😄 

The entry looks good to be merged! 🚀
